use stick_path::functions::errors::CustomError;
use stick_path::run;

fn main() -> Result<(), CustomError> {
    return run();
}
